apps/dashboard-service/src/dashboard/total-section-header.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Get, Query, Controller } from "@nestjs/common";
+import { ApiOperation } from "@nestjs/swagger";
+import { DashboardQueryDto } from "./dto/dashboard-query.dto";
+import { JsonApiResponse } from "@terramatch-microservices/common/decorators";
+import { DelayedJobDto } from "./dto/delayed-job.dto";
+import { CacheService } from "./dto/cache.service";
+import { DelayedJob } from "@terramatch-microservices/database/entities";
+
+@Controller("v3/dashboard/totalSectionHeader")
```

Comment:
Please use plurals for resource paths. Also, this needs a namespace (the bit before `v3`). This should probably be something more like `@Controller("dashboard/v3/totalSectionHeaders")`